src/test/java/org/folio/bulkops/service/PreviewServiceTest.java

```diff
@@ -142,8 +142,8 @@ void shouldReturnPreviewIfAvailable(String fileName, EntityType entityType, Bulk
     } else if (ITEM.equals(entityType)) {
       List<Cell> headers;
       if ((step == EDIT || step == COMMIT) && approachType == ApproachType.IN_APP) {
-        headers = getHeaders(Item.class, Set.of("Binding", "Custom", "Status", "Check Out Notes", "Provenance", "Reproduction", "Check In Notes", "Note", "Administrative Notes"));
-        noteTableUpdater.extendHeadersWithItemNoteTypeNames(ITEM_NOTE_POSITION, headers , List.of("Binding", "Custom", "Note", "Provenance", "Reproduction"), Set.of("Binding","Status","Check Out Notes","Provenance","Check In Notes","Note","Custom", "Reproduction", "Administrative Notes"));
+        headers = getHeaders(Item.class, Set.of("Binding", "Custom", "Status", "Check Out Notes", "Provenance", "Reproduction", "Check In Notes", "Note", "Administrative note"));
+        noteTableUpdater.extendHeadersWithItemNoteTypeNames(ITEM_NOTE_POSITION, headers , List.of("Binding", "Custom", "Note", "Provenance", "Reproduction"), Set.of("Binding","Status","Check Out Notes","Provenance","Check In Notes","Note","Custom", "Reproduction", "Administrative notes"));
```

Comment:
Probably here should be "note", not "notes"?

Reply:
@obozhko-folio 
The changes affect only "Administrative note" - header value
and currently, it's "note", not "notes"